.github/workflows/deno-cd.yml

```diff
@@ -1,26 +1,25 @@
-# This workflow uses actions that are not certified by GitHub.
-# They are provided by a third-party and are governed by
-# separate terms of service, privacy policy, and support
-# documentation.
-
 name: Deno Continuous Deployment
 
 on:
   push:
     paths:
-      - 'src/version.ts'
-    branches: [main]
+      - "src/version.ts"
+    branches:
+      - main
 
 jobs:
   test:
     runs-on: ubuntu-latest
-
+    permissions:
+      contents: write
```

Comment:
📚 https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release

> "Contents" repository permissions (write)

📚 https://github.blog/changelog/2023-11-02-github-actions-enforcing-workflow-scope-when-creating-a-release/

AFAICT releases for this project do not change workflow files or upload artifacts to the release, so the `contents:write` scope is solid!